### DISCUSSION TITLE
Widelands 1.1 and asio dependncy

### DISCUSSION THREAD

**Discussion by muashieWuphu** [widelands/widelands#5749] [Q&A] [answered]
Hi all,
I hope it's an easy question with easy solution.
The linux distro I use doesn't have a separate asio package and asio is provided by boost. The asio.hpp file is so located in /usr/include/boost. How can I tell cmake to search for this file in this location, instead the default /usr/include?

**Comment by Noordfrees** [comment, marked as answer]
Hi,

you can not use Boost's Asio component as a substitute for standalone Asio. They are in separate namespaces and it will not compile.

Please run the script `utils/download_asio.sh`, which will download the Asio sources and deposit them in a directory where CMake will find them.

**Reply by muashieWuphu** [reply]
Thank you very much for the prompt answer! :)
